sd_webui_bayesian_merger/merger.py

```diff
@@ -217,6 +219,9 @@ def merge_block(self, current_bases: Dict, thetas: Dict, key: str) -> Dict:
         alpha = current_bases["alpha"]
         if self.cfg.merge_mode == "weighted_sum":
             return (1 - alpha) * t0 + alpha * t1
+        elif self.cfg.merge_mode == "weighted_subtraction":
+            beta = current_bases["beta"]
+            return (t0 - alpha * beta * t1) / (1 - alpha * beta)
```

Comment:
The use of epsilon should solve this issue. Totally missed that one ;-) I made EPSILON a global variable so it can be used by future methods if required.